package.json

```diff
@@ -5,31 +5,21 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^3.4.2",
     "dedent": "^0.7.0",
     "extract-loader": "^4.0.3",
     "file-loader": "^5.0.2",
     "html-webpack-plugin": "^4.3.0",
     "imports-loader": "^0.8.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^7.0.3",
     "optimize-css-assets-webpack-plugin": "^6.0.1",
-    "postcss": "^8.1",
-    "postcss-loader": "^4.0.3",
     "sass-loader": "^10.3.1",
-    "style-loader": "^1.1.3",
-    "webpack": "^4.41.6",
-    "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^4.11.1",
-    "webpack-merge": "^4.2.2",
-    "webpack-shell-plugin": "^0.5.0",
-    "webpack-watch-files-plugin": "^1.0.3"
+    "webpack": "^5.0.0",
+    "webpack-cli": "^5.0.0"
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "6.1.2",
     "@popperjs/core": "^2.11.6",
-    "bootstrap": "^5.2.2",
-    "optimize-css-assets-webpack-plugin": "^6.0.1"
```

Comment:
I just want to comment here that this dependency (`optimize-css-assets-webpack-plugin`) was not actually removed. But it was duplicated in two both dependency lists, and it really only needs to be in dev dependencies.

For the future: we need to [replace optimize-css-assets Webpack plugin with css-minimizer](https://github.com/NMFR/optimize-css-assets-webpack-plugin).

Reply:
shouldn't we switch to `css-minimizer-webpack-plugin` in this PR though (since in this PR we're switching to webpack 5)?

Reply:
Oh you're right! Somehow I totally missed that. I thought I had seen the bump from 4 to 5 somewhere else. Yes, I agree, we should do that in this PR.